2/6/1/index.css

```diff
@@ -1,89 +1,91 @@
 body {
-    margin: 0 auto;
-    font-family: Arial, Helvetica, sans-serif;
-    width: 900px;
-    background: #EBEBEB;
+  margin: 0 auto;
+  font-family: Arial, Helvetica, sans-serif;
+  width: 900px;
+  background: #ebebeb;
 }
 
 header {
-    display: flex;
-    background-color: #F8AA03;
-    padding: 15px 20px;
+  display: flex;
+  background-color: #f8aa03;
+  padding: 15px 20px;
 }
 
 header a {
-    color: white;
-    font-size: 18px;
-    text-decoration: none;
-    margin-right: 20px;
+  color: white;
+  font-size: 18px;
+  text-decoration: none;
+  margin-right: 20px;
 }
 
-header a:hover, a.active {
-    text-decoration: underline;
+header a:hover,
+a.active {
+  text-decoration: underline;
 }
 
 main {
-    background: white;
-    display: flex;
-    padding: 20px;
+  background: white;
+  display: flex;
+  padding: 20px;
 }
 
 footer {
-    background: #F8AA03;
-    padding: 30px 0;
+  background: #f8aa03;
+  padding: 30px 0;
 }
 
 footer p {
-    color: white;
-    margin: 0;
-    text-align: center;
+  color: white;
+  margin: 0;
+  text-align: center;
 }
 
 h1 {
-    margin: 0;
-    font-size: 21px;
+  margin: 0;
+  font-size: 21px;
 }
 
 h2 {
-    font-size: 19px;
-    margin: 20px 0 10px;
+  font-size: 19px;
+  margin: 20px 0 10px;
 }
 
 h3 {
-    margin: 0;
-    font-size: 17px;
+  margin: 0;
+  font-size: 17px;
 }
 
 img {
-    width: 500px;
-    height: 350px;
+  width: 500px;
```

Comment:
What if we add one more image? I'm not sure why we should define default styles for all images on page.